File: 01_Language/01_Functions/python/lcfirst.py
# coding: utf-8

def lcfirst(s):
    if s is None:
        return ''
    s = str(s)
    if len(s) == 0:
        return ''
    return s[0].lower() + s[1:]

if __name__ == '__main__':
    print(lcfirst('HelloWorld'))
    print(lcfirst('HELLO WORLD'))
    print(lcfirst(' Abc'))
    print(lcfirst(None))

File: 01_Language/01_Functions/js/lcfirst.js
function lcfirst(str) {
    if (str === null || str === undefined) {
        return ''
    }
    return str.charAt(0).toLowerCase() + str.substr(1)
}

console.log(lcfirst('HelloWorld'))
console.log(lcfirst('HELLO WORLD'))
console.log(lcfirst(' Abc'))
console.log(lcfirst(null))
